.github/workflows/builder_nodejs_slsa3.yml

```diff
@@ -83,7 +89,7 @@ jobs:
 
   slsa-publish:
     needs: [slsa-run]
-    if: startsWith(github.ref, 'refs/tags/')
+    if: ${{ inputs.npm-publish }}
```

Comment:
so we don't want to automatically publish on new tags?
Would it complicate because you want to check if the tag is the same as the version declared in the package?
I think it's probably fine to do simply document that users should set to `npm-publish: startsWith(github.ref, 'refs/tags')` instead

Reply:
I set the input's default to be `startsWith(github.ref, 'refs/tags')` so that it should publish by default on new tags.

Reply:
Right now I'm not checking that the tag == the package version. I think that's going to have to be a check that the verifier does anyway.